src/openforms/registrations/contrib/objects_api/tests/test_update_payment_status_v2.py

```diff
@@ -159,6 +169,9 @@ def test_update_payment_status(self):
             public_order_id="TEST-123",
         )
 
+        submission.price = 10.01
+        submission.save()
```

Comment:
direct assignment is not the right approach here, it creates a broken state - a submission cannot have a price if there's no product with price and payment backend setup.

Instead, provide kwargs to the factory:

```py
            form__payment_backend="demo",
            form__product__price=10.01,
```